app/actions/pklima.ts

```diff
@@ -79,7 +77,7 @@ export const changeApprovalTransaction = async (params: {
     params.onStatus("networkConfirmation", "");
     await txn.wait(1);
     params.onStatus("done", "Approval was successful");
-    return value;
+    return formatUnits(value, 9);
```

Comment:
Sure that both should be 18?
Asking because of this comment here:
https://github.com/KlimaDAO/klimadao/blob/87514af8f41548dfda705868911fc18a590e6dab/app/actions/pklima.ts#L35

Reply:
I'd vote for extending this helper here with all possible tokens
https://github.com/KlimaDAO/klimadao/blob/staging/lib/utils/getTokenDecimals/index.ts

and use this everywhere